src/Controller/DownloadController.php

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
This is currently not working as it is hung up on a missing download_base_directory parameter, that seems to not be settable.

Reply:
Also missing download_base_url, Good catch. I'll fix it

Reply:
I'm playing around with the parameterBagInterface, to make things much much more sane imho.